<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(49, 49, 49);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" columnstretch="2,0">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;
color: rgb(79, 163, 167);</string>
          </property>
          <property name="text">
           <string>Diet Restrictions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 170, 255);
font: 700 italic 12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
          </property>
          <item>
           <property name="text">
            <string>NONE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Low Suger</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Low Fat</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Low Cholestrol</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Protein Rich</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fiber Rich</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;
color: rgb(79, 163, 167);</string>
          </property>
          <property name="text">
           <string>Enter the Ingredients Below</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="styleSheet">
           <string notr="true">font: italic 11pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 170, 127);
font: 700 italic 12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Enter Ingredient</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="styleSheet">
           <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
background-color: rgb(85, 255, 255);</string>
          </property>
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" rowspan="4">
       <widget class="QStackedWidget" name="stackedWidget"/>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="styleSheet">
           <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
background-color: rgb(85, 170, 255);</string>
          </property>
          <property name="text">
           <string>Previous</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="styleSheet">
           <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
background-color: rgb(85, 170, 255);</string>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
